docs/user/subprojects.rst

```diff
@@ -1,30 +1,30 @@
-Subprojects
-===========
+Organizing multiple projects in one site: Subprojects
```

Comment:
I'd put the feature name at the front:

```suggestion
Subprojects: host multiple projects on a single domain
```

Reply:
I thought it was so simple and that all words following a colon would be capitalized -- wow I was wrong :upside_down_face: https://www.grammarly.com/blog/capitalization-after-colons/

(also this is correct)